wiki/People/Global_Moderation_Team/en.md

```diff
@@ -70,19 +70,21 @@ Once a Global Moderator chooses to depart from the team, they may be granted the
 
 The [Global Moderation Team group page](https://osu.ppy.sh/groups/4) lists all of the team members. In addition to areas mentioned below, all GMT members take part in [reviewing visual and aural content included in beatmaps](/wiki/Rules/Visual_content_considerations#getting-your-image-assessed).
 
+**A member of the GMT can engage with any tasks within any category that they choose, despite their listed primary responsibilities.** For example, a GMT member primarily responsible for chat moderation may also help with forum moderation, or vice versa.
```

Comment:
```suggestion
**A member of the GMT may engage with any task within any category that they choose, regardless of their listed area of focus.** For example, a GMT member primarily responsible for chat moderation may also help with forum moderation.
```

Reply:
Same wording used as in the [NAT Article](https://osu.ppy.sh/wiki/en/People/Nomination_Assessment_Team#team-members:~:text=A%20member%20of%20the%20NAT%20can%20engage%20with%20any%20tasks%20within%20any%20category%20that%20they%20choose%2C%20despite%20their%20listed%20primary%20responsibilities.%20For%20example%2C%20an%20NAT%20member%20primarily%20responsible%20for%20evaluations%20may%20also%20help%20with%20structural%2Drelated%20tasks%2C%20or%20vice%20versa.). If one is slightly modified, may as well do the other one.

Reply:
can be done in this same pr

Reply:
Will do then